test/assets/input/touches-drag-and-drop.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Drag-and-drop test</title>
+  <style>
+    #drop {
+      width: 5em;
+      height: 5em;
+      border: 1px solid black;
+      position: absolute;
+      top: 30px;
+      left: 0px;
+    }
+
+    #drag {
+      position: absolute;
+      left: 0;
+      top: 0;
+    }
+  </style>
+</head>
+
+<body>
+  <div id="drag" draggable="true">drag me</div>
+  <div id="drop"></div>
+  <script>
+    window.didDragStart = false;
+    window.didDragEnter = false;
+    window.didDragOver = false;
+    window.didDrop = false;
+    const drag = document.getElementById('drag');
+    const drop = document.getElementById('drop');
+    drag.addEventListener('touchstart', function (event) {
```

Comment:
does it actually produce drag events? https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drag_event and related events

Reply:
it is, you can open the mobile mode, the website you paste won't work on mobile mode

Reply:
can we test that? the function is called drag so it'd be nice to ensure that drag events are emitted

Reply:
sorry test what? you mean test the url you paste? or my code? for my code, I use my iphone X and test it on localhost

Reply:
test that the drag event is emitted to the test page.

Reply:
but the drag event you paste only work on mouse, I just test it,

Reply:
the example linked from there https://mdn.github.io/dom-examples/drag-and-drop/copy-move-DataTransfer.html works on my phone. I can long press and drag.

Reply:
you must long press, in that case you can drag

Reply:
that's a little bit different than directly drag

Reply:
I test the url here are some problem I meet,
1. I can't test it if I use mac , open chrome and simulate mobile mode
2. you must long press it and wait a little bit , and than drag, I think everyone one will use other faster command,
let customer press and wait a little bit is really weird unless you design it for other purpose

Reply:
how about I create another pull request ,write another function just for this drag command?

Reply:
and I think when puppeteer start a test case,
it also use chrome and simulate mobile mode,
so the result will same as the first problem I meet, it can't test

Reply:
https://drive.google.com/file/d/1d_eQoI5T-R3A3Xle9SSR5WFiY_C1Psoa/view?usp=sharing

Reply:
https://stackoverflow.com/questions/53530511/imitating-drag-and-drop-events-with-touch-events-for-mobile-devices

Reply:
My point is that the behaviour should be similar to the one for the mouse input in test/src/drag-and-drop.spec.ts 
If the CDP commands we add in the drag and drop don't actually trigger drag and drop, we should not add them.

Reply:
Do you want to reduce the scope of this PR and include press for now? (the only to do there is to use fake timers)

Reply:
maybe we can change the drag to other name like 'tapmove',
I only want to tap the screen and move it.

Reply:
you are right, it can't actually trigger drag and drop in js.